### PR TITLE
Mssw chapter13

### PR DESCRIPTION
**Pull request by Spinodal**

### CODE REVIEW

File: mssw/Animal/lib/Horse.pm [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+sub function1 {
+}
+
+=head2 function2
+
+=cut
+
+sub function2 {
+}
```
Comment (thruddle) [inline mssw/Animal/lib/Horse.pm:51]: Might have been worth cleaning up the placeholder subroutines

File: mssw/Animal/scripts/ch13exercise2.pl [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+use strict;
+use warnings;
+
+use Animal;
+use Cow;
+use Horse;
+use Mouse;
+use Sheep;
+
+use Data::Dumper;
```
Comment (alexdeas) [inline mssw/Animal/scripts/ch13exercise2.pl:12]: Tidy up your dumps!

File: mssw/Animal/lib/Horse.pm (hunk 1 above)
Comment (Spinodal) [inline mssw/Animal/lib/Horse.pm:51]: I actually didn't know they were there!

I couldn't see them through the pod...

File: mssw/Animal/scripts/ch13exercise2.pl (hunk 2 above)
Comment (Spinodal) [inline mssw/Animal/scripts/ch13exercise2.pl:12]: Never!!!

Comment (utopia-2) [inline mssw/Animal/scripts/ch13exercise2.pl:12]: sudo tidy your dumps

**Pull request merged**